aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -589,4 +623,13 @@ private void createRandomObjects(String objectPrefix, int count) {
                     builder -> builder.bucket(s3URI.bucket()).key(s3URI.key() + i).build(),
                     RequestBody.empty()));
   }
+
+
+  /**
+   * A local S3 Express check helper that helps it be used in the annotations for the disabled tests
+   * @return true if S3 Express, false otherwise
+   */
+  private static boolean checkIfS3ExpressHelper() {
+    return S3Express.checkIfS3Express(AwsIntegTestUtil.testBucketName());
```

Comment:
I think we could potentially parameterize the tests, and use `Assume` capability to skip some tests?